tests/kas_with_instance.spec.ts

```diff
@@ -183,3 +183,89 @@ test('create and delete a Kafka Topic', async ({ page }) => {
   await createKafkaTopic(page, testTopicName);
   await deleteKafkaTopic(page, testTopicName);
 });
+
+// test_4kafka.py test_edit_topic_properties_after_creation
+test('edit topic properties after creation', async ({ page }) => {
+  await navigateToKafkaTopicsList(page, testInstanceName);
+  await createKafkaTopic(page, testTopicName);
+
+  const row = page.locator('tr', { hasText: testTopicName });
+  await row.locator('[aria-label="Actions"]').click();
+  await page.getByText('Edit topic configuration').click();
+
+  await expect(page.locator('input[name="num-partitions"]')).toHaveValue('1', { timeout: 3000 });
+  const numPartitionsBefore: string = await page.locator('input[name="num-partitions"]').getAttribute('value');
+  console.log('numPartitionsBefore: ' + numPartitionsBefore);
+  const numPartitionsButton = page.locator('button[name="num-partitions"]');
+  for (let i = 0; i < 2; i++) {
+    await numPartitionsButton.nth(1).click();
+  }
+  await numPartitionsButton.nth(0).click();
+  await expect(page.locator('input[name="num-partitions"]')).not.toHaveValue(numPartitionsBefore);
+
+  // Retention Time
+  await expect(page.locator('label:has-text("days") input[type="number"]')).toHaveCount(1);
+  page.locator('label:has-text("days") input[type="number"]').click();
+  const retentionTimeBefore = await page.locator('label:has-text("days") input[type="number"]').getAttribute('value');
+  const retentionTimeButton = page.locator('button[name="retention-ms"]');
+  for (let i = 0; i < 2; i++) {
+    await retentionTimeButton.nth(1).click();
+  }
+  await retentionTimeButton.nth(0).click();
```

Comment:
Same as above

Reply:
added

Reply:
Can I resolve this conversation @Frawless ?